packages/shared/src/graphql/fragments.ts

```diff
@@ -236,3 +236,12 @@ export const USER_STREAK_FRAGMENT = gql`
     weekStart
   }
 `;
+
+export const SOURCE_CATEGORY_FRAGMENT = gql`
+  fragment SourceCategoryFragment on SourceCategory {
+    id
+    slug
+    title
+    createdAt
```

Comment:
do you need createdAt?
Just be careful with testing this as it might break due to that conversion issue we have.

Reply:
Not at the moment. But yeah probably be safe than sorry. Will remove it quickly.